.github/workflows/stale.yml

```diff
@@ -0,0 +1,27 @@
+name: stale
+
+on:
+  schedule:
+    - cron: '0 23 * * SUN-THU'
+
+jobs:
+  stale:
+
+    runs-on: ubuntu-latest
+
+    if: github.repository == 'optuna/optuna-examples'
+    steps:
+    - uses: actions/stale@v5
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        stale-issue-message: 'This issue has not seen any recent activity.'
+        stale-pr-message: 'This pull request has not seen any recent activity.'
+        close-issue-message: 'This issue was closed automatically because it had not seen any recent activity. If you want to discuss it, you can reopen it freely.'
+        close-pr-message: 'This pull request was closed automatically because it had not seen any recent activity. If you want to discuss it, you can reopen it freely.'
+        days-before-stale: 14
+        days-before-close: 100
```

Comment:
Recently, `optuna` changes the days before stale and close are 7 and 14 for PRs. https://github.com/optuna/optuna/blob/master/.github/workflows/stale.yml Should we apply same strategy here?

Reply:
Thank you for pointing it out. I guess we have no reason to manage different strategies, so I followed `optuna/optuna` in https://github.com/optuna/optuna-examples/pull/119/commits/91e960bb0b4affb09cb8617d516b829eeb795897.